src/components/project/project.service.ts

```diff
@@ -654,14 +658,21 @@ export class ProjectService {
 
   async getRootDirectory(
     projectId: ID,
+    sensitivity: Sensitivity,
     session: Session
   ): Promise<SecuredDirectory> {
     const rootRef = await this.repo.getRootDirectory(projectId, session);
+    const membershipRoles = await this.getMembershipRoles(projectId, session);
+    const permsOfProject = await this.authorizationService.getPermissions({
+      resource: IProject,
+      sessionOrUserId: session,
+      otherRoles: membershipRoles,
+      sensitivity,
+    });
 
-    if (!rootRef) {
+    if (!permsOfProject.rootDirectory.canRead) {
       return {
-        canEdit: false,
-        canRead: false,
+        ...permsOfProject.rootDirectory,
```

Comment:
I think we should just revert this change and the one below. `canEdit` is always false for this property. And for both of these we already know the `canRead` value.